sympy/geometry/ellipse.py

```diff
@@ -1022,6 +1022,35 @@ def auxiliary_circle(self):
         """
         return Circle(self.center, Max(self.hradius, self.vradius))
 
+    def director_circle(self):
+        """Returns a Circle consisting of all points where two perpendicular
```

Comment:
```suggestion
        """
        Returns a Circle consisting of all points where two perpendicular

```
You should have blank lines before and after your headers

Reply:
But it does not matter the code passes the tests

Reply:
Just passing the test is not enough and doctests mostly test examples.

You also need to follow the Documentation and docstring conventions.

It's about making the code readable and convenient, not about passing tests.

Reply:
Also, don't resolve a suggestion/conversation without even talking.

Reply:
sorry for inconvenience, I have changed some code please review